deployment/HDI-Spark/Notebooks/README.md

```diff
@@ -0,0 +1,116 @@
+
+1.2 Select **Spark2**, and then select **LIVY FOR SPARK2 SERVER**.
+
+<img src="../../../docs/img/HDI-Spark-Notebooks/select-livyserver.png" alt="StopLivyServerImage" width="800"/>
+
+1.3 Select **hn0... host**, stop **Livy for Spark2 Server** if started. When prompted, click **OK** to proceed.
+
+- Select hn0 as shown below.
+<img src="../../../docs/img/HDI-Spark-Notebooks/select-host.png" alt="StopLivyServerImage" width="800"/>
+
+- Stop Livy for Spark2 Server.
+<img src="../../../docs/img/HDI-Spark-Notebooks/stop-server.png" alt="StopLivyServerImage" width="800"/>
+
+1.4 Please follow the same step for hn1... host.
+
+## Step 2. Submit HDInsight Script Action
+Return to your HDI cluster in the Azure Portal, and select **Script actions** from the options on the left. You'll submit one script action to deploy SparkDotnet REPL on HDInsight Spark cluster.
+
+Submit Script Action on both Head node and Worker node.
+Run `deploy-sparkdotnet-repl.sh` on the cluster using [HDInsight Script Actions](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-hadoop-customize-cluster-linux):
+
+* Script type: Custom
+* Name: Deploy Sparkdotnet REPL (or anything that is descriptive)
+* Bash script URI: The URI to which you uploaded `deploy-sparkdotnet-repl.sh` (e.g. https://<my storage account>.blob.core.windows.net/<my container>/<some dir>/deploy-sparkdotnet-repl.sh)
+* Node type(s): Head and Worker
+* Parameters: Sparkdotnet version. For example, if you want to install Sparkdotnet version 0.6.0 then it would be `0.6.0`.
+
+The following captures the setting for a HDInsight Script Action on Head node:
+
+<img src="../../../docs/img/HDI-Spark-Notebooks/deploy-sparkdotnet-repl-scriptaction.png" alt="ScriptActionImage" width="500"/>
+
+After you see green check-marks next to the status of both script actions, you can move to the next step.
+
+## Step 3. Start Livy Server
+Please follow the above [Step 1](#step-1-stop-livy-server) to **Start** Livy for Spark2 Server.
+
+## Step 4. Set up Spark Default Configs
+4.1 From the portal, select **Overview**, and then select **Ambari home**. If prompted, enter the cluster login credentials for the cluster.
+
+4.2 Select **Spark2**, and then Select **CONFIGS**, then click on **Custom spark2-defaults**.
+
+<img src="../../../docs/img/HDI-Spark-Notebooks/spark-configs.png" alt="SetConfigsImage" width="800"/>
+
+4.3 Click on **Add Property...** to add Spark default settings.
+You will add a total of 4 individual properties. Add them 1 one at a time, all using **TEXT** Property Type in Single property add mode. Make sure you don’t have any extra spaces before or after any of the keys/values.
+- Property 1:
+    - Key:&ensp;&ensp;`spark.dotnet.shell.command`
+    - Value: `/usr/share/dotnet-tools/dotnet-try,kernel-server,--default-kernel,csharp`
+- Property 2:
+    - Key:&ensp;&ensp;`spark.dotnet.packages`
+    - Value: `["nuget: Microsoft.Spark, 0.5.0", "nuget: Microsoft.Spark.Extensions.Delta, 0.5.0"]`
```

Comment:
Do you want to update this version # to 0.6.0 ?

Reply:
Done